bin/index.js

```diff
@@ -1,141 +1,9 @@
-        return process.exit(1)
-      } else {
-        return val
-      }
-    },
-    initial: 'My Blits App', // Default value for the application name
-  },
-  {
-    type: 'text',
-    name: 'appPackage',
-    message: 'What is the package name of your App?',
-    format: val => {
-      // Validate the package name using validate-npm-package-name
-      if (!validatePackage(val).validForNewPackages) {
-        spinnerMsg.fail(red(bold("Please provide a valid package name")))
-        return process.exit(1)
-      } else {
-        return val
-      }
-    },
-    initial: prev => `${prev.replace(/[\sA-Z]/g, str => str === ' ' ? '-' : str.toLowerCase())}`,
-  },
-  {
-    type: 'text',
-    name: 'appFolder',
-    message: 'In what folder (relative to the current location) should the App be generated?',
-    format: (val, prev) => {
-      // Regular expression to validate whether the path is Unix/Windows-based
-      const pathRegex = /([A-Za-z]:)?([\\/]+\w+)+/
-      // Check if the provided path matches the defined regex
-      if (pathRegex.test(val)) {
-        try {
-          // Check if the path exists
-          if (fs.statSync(val)) {
-            // Return the resolved file path using path.join
-            return `${path.join(val, prev.appPackage)}`
-          }
-        } catch (e) {
-          // Handle case where an error occurred during file system interaction
-          if (e.code === 'ENOENT') {
-            spinnerMsg.fail(red(bold("Entered directory path is invalid, please enter a valid directory!")))
-            process.exit()
-          }
-        }
-      } else if (val === prev.appPackage) {
-        return path.join(process.cwd(), prev.appPackage)
-      }
-    },
-    initial: (prev, val) => val.appPackage,
-  },
-  {
-    type: 'toggle',
-    name: 'esLint',
-    message: 'Do you want to enable eslint?',
-    initial: 'true',
-    active: 'Yes',
-    inactive: 'No',
-  },
-  {
-    type: 'toggle',
-    name: 'gitInit',
-    message: 'Do you want to generate an git repository?',
-    initial: 'true',
-    active: 'Yes',
-    inactive: 'No',
-  },
-]
-
-
-const createApp = () => {
-  return new Promise(resolve => {
-    let config
-    sequence([
-      async () => {
-        const onCancel = () => {
-          process.exit()
-        }
-        config = await prompts(questions, { onCancel })
-        config.fixturesBase = fixturesBase
-        return config
-      },
-      () => {
-        spinnerMsg.start(`Generating new App "${config.appName}" ...`)
-        copyLightningFixtures(config).then(targetDir => (config.targetDir = targetDir))
-        spinnerMsg.succeed()
-      },
-      () => setAppData(config),
-      () => setBlitsVersion(config),
-      () => config.esLint && addESlint(config),
-      () => config.gitInit && gitInit(config.targetDir, config.fixturesBase),
-      () => done(config)
-    ])
-  })
+try {
+  execSync('npx --yes blits-create@latest', { stdio: 'inherit' })
```

Comment:
Once after publishing to the real location we can update the location here(might be like @lightningjs/blits-create@latest